web/content/release-nightly.adoc

```diff
@@ -0,0 +1,71 @@
+:FOREMAN_VER: nightly
+:KATELLO_VER: nightly
+
+= Foreman {FOREMAN_VER} and Katello {KATELLO_VER}
+
+Use the top menu bar for navigation in the following guides:
+
+* link:/{FOREMAN_VER}/Planning_Guide/index-foreman-el.html[Planning for Foreman]
+* link:/{FOREMAN_VER}/Quickstart_Guide/index-foreman-el.html[Foreman Quickstart Guide on RHEL/CentOS]
+* link:/{FOREMAN_VER}/Quickstart_Guide/index-foreman-deb.html[Foreman Quickstart Guide on Debian/Ubuntu]
+* link:/{FOREMAN_VER}/Quickstart_Guide/index-katello.html[Katello Quickstart Guide on RHEL/CentOS]
+* link:/{FOREMAN_VER}/Installing_Server_on_Red_Hat/index-foreman-el.html[Installing Foreman on RHEL/CentOS]
+* link:/{FOREMAN_VER}/Installing_Server_on_Red_Hat/index-katello.html[Installing Katello on RHEL/CentOS]
+* link:/{FOREMAN_VER}/Installing_Proxy_on_Red_Hat/index-foreman-el.html[Installing Smart Proxy on RHEL/CentOS]
+* link:/{FOREMAN_VER}/Installing_Server_on_Debian/index-foreman-deb.html[Installing Foreman on Debian/Ubuntu]
+* link:/{FOREMAN_VER}/Installing_Proxy_on_Debian/index-foreman-deb.html[Installing Smart Proxy on Debian/Ubuntu]
+* link:/{FOREMAN_VER}/Upgrading_and_Updating/index-foreman-el.html[Upgrading and Updating Foreman]
+* link:/{FOREMAN_VER}/Upgrading_and_Updating/index-katello.html[Upgrading and Updating Katello]
+* link:/{FOREMAN_VER}/Deploying_on_AWS/index-foreman-el.html[Deploying Foreman on AWS (reference guide)]
+* link:/{FOREMAN_VER}/Configuring_Load_Balancer/index-foreman-el.html[Configuring Smart Proxies with a Load Balancer]
+* link:/{FOREMAN_VER}/Provisioning_Guide/index-foreman-el.html[Provisioning Guide]
+* link:/{FOREMAN_VER}/Content_Management_Guide/index-katello.html[Content Management Guide]
+* link:/{FOREMAN_VER}/Configuring_Ansible/index-foreman-el.html[Configuring Foreman to use Ansible]
+* link:/{FOREMAN_VER}/Managing_Hosts/index-foreman-el.html[Managing Hosts Guide]
+* link:/{FOREMAN_VER}/Administering_Red_Hat_Satellite/index-foreman-el.html[Administering Foreman]
+* link:/{FOREMAN_VER}/Application_Centric_Deployment/index-foreman-el.html[Application Centric Deployment]
```

Comment:
Do we really need `link:` here? What's with the first `/` ?

See also https://docs.asciidoctor.org/asciidoc/latest/macros/link-macro/

Reply:
This is needed when you run the site locally and want to test it. I did not think about it deeply, what would you like to see there? Relative URL instead?

Reply:
If it works for both our website as well as local tests, then I'm fine with it. Please add a short comment and why it's like this.